chapters/annotations.tex

```diff
@@ -42,6 +42,7 @@ \section{Documentation}\label{annotations-for-documentation}\label{documentation
   String info = "" "Description of the class";
```

Comment:
When thinking about the `styleSheets` description string, it strikes me that it seems wrong to describe `into` as a `description`.  Shouldn't we say _Documentation of the class_ to avoid confusion with the class' description string?
```suggestion
  String info = "" "Documentation of the class";
```

Reply:
That line wasn't changed so I don't think we should change it here. Especially as that would lead to something described as Documentation inside the Documentation-annotation, which both looks a bit odd and can lead to ambiguities as Documentation can then be seen as referring to either of them.

Both info and revisions document _aspects_ of the class; it may be that we can find a better description, preferably including "Information"  - but in another PR.

Reply:
Sure.  Resolving.